bpf/aya-bpf/src/programs/tracepoint.rs

```diff
@@ -0,0 +1,20 @@
+use crate::{helpers::bpf_probe_read, BpfContext};
+use core::ffi::c_void;
+
+pub struct TracePointContext(*mut c_void);
```

Comment:
please make the ctx pointer a namd field for consistency with the other contexts